modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/CTCollectionResourceImpl.java

```diff
@@ -349,7 +353,10 @@ private DefaultDTOConverterContext _getDTOConverterContext(
 			).put(
 				"publish",
 				() -> {
-					if (!_isPublishEnabled(ctCollection.getCtCollectionId())) {
+					if (!_isPublishEnabled(ctCollection.getCtCollectionId()) ||
+						(ctCollection.getStatus() ==
+							WorkflowConstants.STATUS_EXPIRED)) {
```

Comment:
The status check should be added to `_isPublishEnabled` to consolidate the logic.

I think we should restrict the publish/schedule actions only to publications that are in `STATUS_DRAFT`. @david-truong does this make sense? or should we allow other statuses to publish?

Whatever we decide, the same check should be apply to [CTCollectionService#publishCTCollection](https://github.com/liferay/liferay-portal/blob/master/modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTCollectionServiceImpl.java#L188-L191)